src/Switcher/Switcher.tsx

```diff
@@ -20,6 +20,12 @@ const Switcher: React.FC<SwitcherProps> = ({ selected, onChange, ...rest }) => {
     return value === selected;
   };
 
+  const getSelectedIndex = () => {
+    const options = rest.children.map((child) => child.props.value);
+
+    return options.indexOf(selected);
```

Comment:
```suggestion
    return React.Children.toArray(rest.children).findIndex((child, index) =>
      isSelected((child as ReactElement)?.props?.value, index)
    )
```

Reply:
This looks great to me, I didn't know about `React.Children.toArray`